models/transform/orders_calculations.sql

```diff
@@ -61,8 +61,8 @@ order_numbers as (
         *,
         
         case
-            when completed_order_number = 1
-                then 'new'
+            when completed_order_number is null then null
+            when completed_order_number = 1 then 'new'
```

Comment:
For whatever reason, this makes more sense to me in reverse order
```sql
case
    when completed_order_number = 1 then 'new'
    when completed_order_number is not null then 'repeat'
    else null -- this line is optional, since "else null" is always implied
end as new_vs_repeat
```

Reply:
love it! thanks!